### DISCUSSION TITLE
Submenu Items

### DISCUSSION THREAD

**Discussion by diogovitzo** [lucky-media/landtamic#23] [General]
Hi, 

Recently purchased Landtamic so it would make our efforts to migrate our product's website to Statamic, and so far so good. 

However, one thing I can't seem to figure out is how to display submenu nav items. Is this supported at all?

Thanks!

**Comment by lokmanm** [comment]
Hey @diogovitzo!

We don't support submenus because there are tons of variants, and it'd be a huge hassle to maintain all the possible variations.

We'll remember this when we're rolling out a major upgrade, and I'll give you a heads-up if we decide to go for it.

Thanks!

**Comment by diogovitzo** [comment]
Hi,

In that case, I would suggest that you [remove the "More" navbar item](https://i.imgur.com/ck3QbQM.png) from all your materials as it misleads customers into thinking this is supported.

**Reply by lokmanm** [reply]
Hey @diogovitzo!

You're right, that's misleading and I apologize. I'll have someone from the team update the content.

cc @artzenki

**Reply by artzenki** [reply]
Hello @diogovitzo, 

Thanks for informing us, the website is now up to date with the removed `More` submenus.

Wish you all the best!

**Reply by diogovitzo** [reply]
No worries. In the meantime I managed to develop the submenu so we're able to proceed with minor harm 👍 

Thanks for Landtamic! It is far from perfect, but it provides a solid foundation for those starting out with Statamic.
